package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dashevo/dpp",
-  "version": "0.17.0-dev.4",
+  "version": "0.17.0-dev.5",
   "description": "The JavaScript implementation of the Dash Platform Protocol",
```

Comment:
Don't we need to update dashcore-lib version?

Reply:
It was updated in https://github.com/dashevo/js-dpp/pull/278